### PR TITLE
Add Base mainnet (8453)

### PR DESCRIPTION
**Pull request by mdehoog**
# Add New Chain 8453

## Checklist

- [x] The branch is named as `add-chain-<chainId>`.
- [x] I haven't modified the [chains.json](../../src/chains.json) file directly.
- [x] In [sourcify-chains.ts](../../src/sourcify-chains.ts) file
  - [x] I've set `supported: true`.
  - [x] I've set `monitored: false`.
  - [x] I haven't added an `rpc` field but the one in [chains.json](../../src/chains.json) is used (if not, please explain why).
- [x] I've added a test in [chain-tests.js](../../test/chains/chains-test.js) file.
- [ ] `test-new-chain` test in Circle CI is passing.

### CODE REVIEW

File: src/sourcify-chains.ts [hunk 1]
```diff
@@ -793,6 +793,14 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
     contractFetchAddress: "https://goerli.basescan.org/" + ETHERSCAN_SUFFIX,
     txRegex: ETHERSCAN_REGEX,
   },
+  "8453": {
+    // Base Mainnet
+    supported: true,
+    monitored: false,
+    contractFetchAddress: "https://basescan.org/" + ETHERSCAN_SUFFIX,
+    rpc: ["https://developer-access-mainnet.base.org"],
```
Comment (mdehoog) [inline src/sourcify-chains.ts:801]: can remove this line once `chains.json` is regenerated (now that https://github.com/ethereum-lists/chains/pull/3165 is merged)

**Comment by kuzdogan** [comment]
Updated chains.json. Can you please remove the `rpc` and rebase/merge from staging?

**Comment by mdehoog** [comment]
Any suggestions on getting tests to pass?
```
Error: No RPC provider was given for this chain with id 1 and name Ethereum Mainnet
```
Locally this was resolved by copying `.env.dev` to `.env` but don't want to commit that.

**Comment by kuzdogan** [comment]
I don't know where this has come from. I'll have a look

**Comment by mdehoog** [comment]
@kuzdogan this is caused by https://github.com/ethereum/sourcify/commit/f4d68cf8fceb807598550a8d185ebf8ddca315d4

**Comment by kuzdogan** [comment]
This was unexpected because the tests were passing on our commits. On external PRs there are no env vars and the `rpc` of those chains with env var rpcs end up as an empty array, hence the error. 

Can you please rebase one last time?

**Pull request merged**